jmx_prometheus_javaagent/src/test/java/io/prometheus/jmx/ArgumentsTest.java

```diff
@@ -0,0 +1,333 @@
+                "[001:0db8:0a0b:12f0:0000:0000:0000:0001]:12345:/opt/prometheus/config.yaml",
+                Arguments.Mode.HTTP,
+                "001:0db8:0a0b:12f0:0000:0000:0000:0001",
+                12345,
+                "/opt/prometheus/config.yaml"),
+        new ArgumentsTestDefinition(
+                VALID_CONFIGURATION,
+                "[001:0db8:0a0b:12f0:0000:0000:0000:0001]:12345:/opt/prometheus/config_file.yaml",
+                Arguments.Mode.HTTP,
+                "001:0db8:0a0b:12f0:0000:0000:0000:0001",
+                12345,
+                "/opt/prometheus/config_file.yaml"),
+        new ArgumentsTestDefinition(
+                VALID_CONFIGURATION,
+                "[001:0db8:0a0b:12f0:0000:0000:0000:0001]:12345:/opt/prometheus/config-file.yaml",
+                Arguments.Mode.HTTP,
+                "001:0db8:0a0b:12f0:0000:0000:0000:0001",
+                12345,
+                "/opt/prometheus/config-file.yaml"),
+        new ArgumentsTestDefinition(
+                VALID_CONFIGURATION,
+                "[001:0db8:0a0b:12f0:0000:0000:0000:0001]:12345:/opt/prometheus/jmx-exporter/config-file.yaml",
+                Arguments.Mode.HTTP,
+                "001:0db8:0a0b:12f0:0000:0000:0000:0001",
+                12345,
+                "/opt/prometheus/jmx-exporter/config-file.yaml"),
+        new ArgumentsTestDefinition(
+                VALID_CONFIGURATION,
+                "[001:0db8:0a0b:12f0:0000:0000:0000:0001]:12345:/opt/prometheus/jmx_exporter/config-file.yaml",
+                Arguments.Mode.HTTP,
+                "001:0db8:0a0b:12f0:0000:0000:0000:0001",
+                12345,
+                "/opt/prometheus/jmx_exporter/config-file.yaml"),
+        new ArgumentsTestDefinition(
+                VALID_CONFIGURATION,
+                "[001:0db8:0a0b:12f0:0000:0000:0000:0001]:12345:/opt/prometheus/jmx_exporter/config-file.yaml",
+                Arguments.Mode.HTTP,
+                "001:0db8:0a0b:12f0:0000:0000:0000:0001",
+                12345,
+                "/opt/prometheus/jmx_exporter/config-file.yaml"),
+        new ArgumentsTestDefinition(
+                VALID_CONFIGURATION,
+                "/opt/prometheus/config.yaml",
+                Arguments.Mode.OPEN_TELEMETRY,
+                null,
+                null,
+                "/opt/prometheus/config.yaml"),
+        /*
+        new ArgumentsTestDefinition(
+                INVALID_CONFIGURATION,
+                "::/opt/prometheus/config.yaml",
+                Arguments.Mode.HTTP,
+                null,
+                null,
+                "/opt/prometheus/config.yaml"),
+        new ArgumentsTestDefinition(
+                INVALID_CONFIGURATION,
+                "::/opt/prometheus/config.yaml",
+                Arguments.Mode.OPEN_TELEMETRY,
+                null,
+                null,
+                "/opt/prometheus/config.yaml")*/
+    };
+
+    @Test
+    public void testsArguments() {
```

Comment:
parmeterized test is easier to troubleshoot

see https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-MethodSource
and https://github.com/grafana/grafana-opentelemetry-java/blob/50411a43bf63896d4f68f1634d5a047e365399d2/custom/src/test/java/com/grafana/extensions/resources/ResourceCustomizerTest.java for an example

Reply:
Fixed.